### PR TITLE
Technical: fix kubernetes resources

### PR DESCRIPTION
**Pull request by orifito**

### CODE REVIEW

File: newspaper-api.yml [hunk 1]
```diff
@@ -22,20 +22,25 @@ metadata:
 spec:
   ports:
     # the port that this service should serve on
-  - port: 38765
+  - port: 80
```
Comment (mychkin) [inline newspaper-api.yml:25]: did you check that's 80?  where can I check that?

File: newspaper-api.yml [hunk 2]
```diff
@@ -45,7 +50,7 @@ spec:
     spec:
       containers:
       - name: master
-        image: smarp/newspaper-api
+        image: smarp/newspaper-api:v2.0
```
Comment (mychkin) [inline newspaper-api.yml:53]: now we will need to manually specify tag, right?

File: newspaper-api.yml (hunk 1 above)
Comment (orifito) [reply newspaper-api.yml:25]: yes sure... change your kubectl context to `smarp-cluster` and run `kubectl describe service newspaper-api`

File: newspaper-api.yml (hunk 2 above)
Comment (orifito) [reply newspaper-api.yml:53]: no, it's not something new; actually I find it really strange that it wasn't set...

Comment (mychkin) [reply newspaper-api.yml:53]: right

**Pull request merged**